lib/ansible/module_utils/facts.py

```diff
@@ -2221,6 +2221,12 @@ class LinuxNetwork(Network):
     - ipv4_address and ipv6_address: the first non-local address for each family.
     """
     platform = 'Linux'
+    _interface_type = {
```

Comment:
looks like other similar maps in facts tends to use constant style naming
(ie, INTERFACE_TYPE).